src/platforms/common/configuration/options.mdx

```diff
@@ -536,15 +536,35 @@ Configures a separate proxy for outgoing HTTPS requests. This value might not be
 
 </ConfigKey>
 
-<ConfigKey name="cache-directory-path" supported={["dotnet", "android"]}>
+<ConfigKey name="cache-directory-path" supported={["dotnet", "android", "unity"]}>
 
 Specifies a local directory used for caching payloads. When this option is enabled (that is, when the directory is set), the Sentry SDK will persist envelopes locally before sending to Sentry. This configuration option is particularly useful if you expect your application to run in environments where internet connectivity is limited.
 
+</ConfigKey>
+
+<PlatformSection supported={["android"]}>
+
+Default: set to `android.content.Context.getCacheDir()/sentry`.
+
+</PlatformSection>
+
+<PlatformSection supported={["dotnet"]} notSupported={["dotnet.xamarin", "dotnet.maui", "unity"]}>
+
 Default: not set (caching is disabled).
 
-Client SDKs built on top of the Sentry SDK for .NET, such Xamarin and Unity, have this feature enabled by default. For example in Unity, the default value is `Application.persistentDataPath`. You can disable offline caching by setting it to `null`.
+</PlatformSection>
+
+<PlatformSection supported={["unity"]}>
 
-</ConfigKey>
+The default value is `Application.persistentDataPath`. You can disable offline caching by setting it to `null`.
+
+</PlatformSection>
+
+<PlatformSection supported={["dotnet.xamarin", "dotnet.maui"]}>
+
+The default value is `Environment.GetFolderPath(Environment.SpecialFolder.LocalApplicationData));`
```

Comment:
It's different for MAUI.

https://github.com/getsentry/sentry-dotnet/blob/f712c72d2b2b941a2b098ec68401d14e7eb3520a/src/Sentry.Maui/Internal/SentryMauiOptionsSetup.cs#L36

I'll send a corrected PR.

Reply:
See #5519